### PR TITLE
aws-nuke 3.22.0

### PR DESCRIPTION
**Pull request by daeho-ro**
Fix formula for new version.
This will close below PRs too.
- https://github.com/Homebrew/homebrew-core/pull/190893
- https://github.com/Homebrew/homebrew-core/pull/191057

----

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/a/aws-nuke.rb [hunk 1]
```diff
@@ -20,29 +20,16 @@ class AwsNuke < Formula
   depends_on "go" => :build
 
   def install
-    build_xdst="github.com/ekristen/aws-nuke/v#{version.major}/cmd"
-    ldflags = %W[
-      -s -w
-      -X #{build_xdst}.BuildVersion=#{version}
```
Comment (carlocab) [inline Formula/a/aws-nuke.rb:26]: Does
```
aws-nuke --version
```
still produce the right output after dropping this?

Comment (daeho-ro) [reply Formula/a/aws-nuke.rb:26]:
```
> aws-nuke --version
aws-nuke version 3.0.0-dev
```
It seems that the version is hard-coded.

Comment (daeho-ro) [reply Formula/a/aws-nuke.rb:26]: OK, similaly as before, I think I can add the version manually.
It is now return the correct one.

```
> aws-nuke --version
aws-nuke version 3.22.0
```

File: Formula/a/aws-nuke.rb [hunk 2]
```diff
@@ -20,29 +20,27 @@ class AwsNuke < Formula
...
     end
 
-    pkgshare.install "config"
+    pkgshare.install "pkg/config"
 
     generate_completions_from_executable(bin/"aws-nuke", "completion")
   end
 
   test do
     assert_match "InvalidClientTokenId", shell_output(
```
Comment (carlocab) [inline Formula/a/aws-nuke.rb:40]:
```suggestion
  test do
    assert_match version.to_s, shell_output("#{bin}/aws-nuke --version")
    assert_match "InvalidClientTokenId", shell_output(
```
Let's make sure this doesn't regress.

**Review by carlocab — APPROVED** [review]
Thanks; one small comment!

**Review by carlocab — APPROVED** [review]

**Pull request merged**